system/settings/settings.xml

```diff
@@ -1790,7 +1790,7 @@
           <control type="spinner" format="integer" />
         </setting>
         <setting id="services.esallinterfaces" type="boolean" label="14277" help="36338">
-          <level>1</level>
+          <level>3</level>
```

Comment:
Can we take care here just how confusing and/or diffcult this is going to make it for users to configure and use remote apps (like Kore, Official iOS Remote, Yatse etc.) to control Kodi.

>Users should not usually enable this setting, so elevate the setting level to expert.

Controlling Kodi with a remote app is pretty normal use, at least there are millions of users that do so and they are far from all "experts". Hiding the setting and making using it sound scary/dangerous is just going to increase the number of user support requests we get.